packages/cbioportal-clinical-timeline/src/TimelineStore.tsx

```diff
@@ -31,9 +30,60 @@ export class TimelineStore {
         events: TimelineEvent[];
         index: number;
     };
+    @observable groupByOption: Readonly<string> | null = null;
```

Comment:
i would like to move all this stuff out of TimelineStore, which should be generic to all timelines.   This stuff should probably be in a new store that belongs to TimelineWrapper or in component state of top level VAF.

Reply:
Fixed.